test/screens/play/play_page_test.dart

```diff
@@ -37,6 +39,36 @@ void main() {
     expect(find.byType(PlayerHand), findsOneWidget);
   });
 
+  testWidgets('should not display the game over page with no game result', (WidgetTester tester) async {
+    final state = makeGameState();
+    state.nextCard();
+
+    await tester.pumpWidget(createWidgetForTesting(child: PlayPage('Play Page', state)));
+
+    expect(find.byType(GameEndPage), findsNothing);
+    expect(find.byType(PlayerHand), findsOneWidget);
+  });
```

Comment:
I know I suggested it, but this is basically a C+P of the test above so doesn't really add much. Perhaps we just add the `expect(find.byType(GameEndPage), findsNothing);` to the previous test and update the description, like: "should display the regular page if game is in progress and no cards being scryed" ?